mix.exs

```diff
@@ -30,12 +33,11 @@ defmodule Kitto.Mixfile do
      {:plug, "~> 1.2"},
      {:poison, "~> 3.0"},
      {:fs, github: "kittoframework/fs", tag: "v1.9-kitto-1"},
-     {:ex_doc, ">= 0.0.0", only: :dev},
+     {:ex_doc, "~> 0.14", only: :dev},
```

Comment:
No need to unlock them all at the time. Not enough time to go through the changelog of each of the updated deps.

Run `mix deps.get` and commit the changes inflicted by your updates in `mix.exs`.